narratives/ncov_sit-rep_2020-05-15.md

```diff
@@ -0,0 +1,220 @@
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 2-->
+# [COVID-19 Resources](https://nextstrain.org/ncov/global/2020-05-14?d=tree&p=full&legend=closed)
+
+#### Nextstrain Resources
+* [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).
+* [Previous Situation Reports](https://nextstrain.org/ncov-sit-reps/).
+* [Explanatory Twitter threads](https://bedford.io/misc/twitter/).
+* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
+* [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).
+
+#### External Resources
+* [How coronavirus mutations and spreads (NYTimes)](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html).
+* [Ask a Scientist & FAQs](https://covid19.fas.org/l/en).
+* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
+* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
+
+<!-- This is the right-side text -->
+```auspiceMainDisplayMarkdown
+
+# Situation Report Hiatus
+<p>
+Early in the pandemic, it was unclear how SARS-CoV-2 was moving between countries, where it was circulating locally, and how localized outbreaks were related.
+Over the last four months, we've tackled these questions and more in our weekly Situation Reports. At this stage of the pandemic, it's clear that these broad-strokes patterns are largely consistent across regions and countries:  
+</p>
+<p/>
+
+- Outbreaks across even distant parts of the world are deeply intertwined.<p/>
+
+- Through human migration and travel, the virus has been introduced to most communities multiple times.<p/>  
+
+- Once these "sparks" land in a new community, many fizzle out without causing widespread transmission. Subject to local conditions and a bit of chance, some of these sparks grow into local outbreaks.<p/>  
+
+- Eventually, these local outbreaks send of sparks of their own, spreading to new locations.  
+<p/>
+
+Unsurprisingly, we see this pattern in countries experiencing their first wave of infections. More alarmingly, we also see this pattern after reintroduction to  countries where the initial peak passed months beforehand.
+Ultimately, this pattern is only broken when a country is able to effectively test, trace, and isolate cases immediately.
+<br><br>
+
+This means that outbreaks across the world are deeply intertwined, and the battle against COVID-19 will always be global -- we can't conquer the virus anywhere without addressing it everywhere.
+<br><br>
+
+But, at this stage of the pandemic, genomic epidemiology is most urgently needed within hyperlocal communities through local public health offices.
+<br><br>
+
+As such, this will be our last weekly Situation Report for awhile as we shift our focus to supporting local public health officials.
```

Comment:
Just a _little_ unsure about this wording as I feel like this may lay more of a groundwork than we intend to. While we're certainly supporting local public health officials, I'm not sure I'd like to imply it's our new focus (or that we have only one focus!). 

Maybe just changing slightly to "...our last weekly Situation Report for a while, though we aim to support similar reports from local public health officials." ?